modules/create_timepoint/jsx/createTimepointIndex.js

```diff
@@ -0,0 +1,356 @@
+import React from 'react';
+import Panel from 'Panel';
+import Loader from 'Loader';
+import PropTypes from 'prop-types';
```

Comment:
i would group react and proptypes together, then panel and loader together to make it clear what's a react package and what's a loris component